### PR TITLE
Add docs for Guardian `valve` implementation

### PR DESCRIPTION
**Pull request by bachya**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/107423
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/guardian.markdown [hunk 1]
```diff
@@ -31,7 +33,8 @@ There is currently support for the following device types within Home Assistant:
 - **Binary sensor**: reports the status of the onboard leak detector and access point
 - **Button**: add various configuration controls
 - **Sensor**: reports on the device's detected temperature and uptime
-- **Switch**: allows the user to open and close the valve
+- **Switch**: allows the user to enable and disable the onboard access point
+- **Valve**: allows the user to open and close the valve (deprecated)
```
Comment (bdraco) [inline source/_integrations/guardian.markdown]: I think the (deprecated) comment is on the wrong place?

Comment (bachya) [reply source/_integrations/guardian.markdown]: Ah, yes. Based on your comments in https://github.com/home-assistant/core/pull/107423, should I mention deprecation at all?

Comment (bdraco) [reply source/_integrations/guardian.markdown]: Probably too soon so I'd take it out for now

**Review by bdraco — APPROVED** [review]

**Comment by bachya** [comment]
Thanks!

**Pull request merged**